spyder_kernels/utils/nsview.py

```diff
@@ -292,7 +292,7 @@ def default_display(value, with_module=True):
         return type_str[1:-1]
 
 
-def collections_display(value, level):
+def collections_display(value, level, numeric_value=False):
```

Comment:
I don't think it's necessary to add `numeric_value` as a kwarg to this function because its purpose is to return a string representation of a collection.